tests/test_txlist.py

```diff
@@ -54,8 +35,8 @@ def txlist(b, user_pk, user2_pk, user_sk, user2_sk, genesis_block):
 @pytest.mark.bdb
 def test_get_txlist_by_asset(b, txlist):
     res = b.get_transactions_filtered(txlist.create1.id)
```

Comment:
[`get_transaction_filtered`](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/lib.py#L284) uses `status`, I think we can skip that and directly yield i.e.